src/Core/src/Platform/Windows/ImageSourcePartExtensions.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.Maui.Platform
 {
-	internal static class ImageSourcePartExtensions
+	public static class ImageSourcePartExtensions
```

Comment:
Feedback on making these methods public for .NET 8? We must also add a property to allow set the ImageSource of the ImageSourcePartLoaded?